client/test/README.md

```diff
@@ -0,0 +1,46 @@
+## Single test runner
+
+Single test runner loads only test files that are listed in config.json file. Configuration reflects folders structure.
+
+### Extending config.json
+
+Example folder: `client/state/plugins/wporg/test/`
+
+Test files we want to add to the runner:
+* `reducer.js`
+* `test-actions`
+* `test-selectors`
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/commit/4c13e0fd5f9d3a3c939dd7d1ba969cb06b0fcad8 :)